app/models/cake.rb

```diff
@@ -0,0 +1,5 @@
+class Cake < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true, length: { minimum: 3, maximum: 30 }
+  validates :kind, presence: true, inclusion: { in: %w(tasty non-gluten vegan), message: "%{value} is not a valid kind" }
```

Comment:
'The default error message for this helper is "is not included in the list".' 
http://guides.rubyonrails.org/active_record_validations.html#inclusion